README.md

```diff
@@ -8,8 +8,10 @@ The project uses poetry for Python to create an isolated environment and manage
 
 ### Poetry installation (Bash)
 
+Note you will need to install Python for this.
```

Comment:
I had to set this up on a new laptop so added some steps to the README where I had issues